docs/websites/nginx/nginx-php-fpm-debian-8.md

```diff
@@ -0,0 +1,165 @@
+High performance Nginx and PHP over Debian 8
+19th May 2015 by Javier Briz
+
+
+This document describes how to install and setup an nginx server with PHP-FPM.
+This stack, together with a MySQL server, is what is called a **LEMP** server. After following this article, your PHP code will be
+able to connect to a MySQL server, but the MySQL server installation itself will not be explained;
+there are already [lots of guides](https://www.linode.com/docs/databases/mysql/) on this topic. 
+
+#Installation Prerequisites
+First of all, follow the [Getting Started Guide](https://www.linode.com/docs/getting-started/).
+It is also usefull to have a Fully Qualified Domain Name to have a handy pointer to the server.
+
+#Packages installation
+In this step, we are installing all the necessary packages.
+Run the following as root to install nginx, PHP and MySQL connector:
+
+        apt-get update
+        apt-get install nginx php5-fpm php5-mysql
+
+Make sure nginx and PHP-FPM are running
+
+        ps aux|grep -e nginx -e php
+        
+#Nginx setup
+Now we have to setup nginx server to relay on PHP-FPM for PHP requests. 
+By default, PHP-FPM on debian listens on a socket. We are putting everything in one single server so this is ok.
+To find the path for the socket run:
+
+        cat /etc/php5/fpm/pool.d/www.conf |grep 'listen ='
+
+Probably, it is located in `/var/run/php5-fpm.sock`
+
+We will tell nginx where is this socket in a *server* context in the nginx setup.
+In other words, add following lines to `/etc/nginx/sites-enabled/default`, below start of *server* section:
```

Comment:
More clarification on where to put the location block is needed here. Either by mentioning line numbers, or showing a small snippet of what would come before and after where this block needs to be placed.